test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Enums/EnumsController.cs

```diff
@@ -245,4 +246,24 @@ public ITestActionResult HasAccessLevel([FromODataUri] int id, [FromODataUri] Ac
             return Ok(result);
         }
     }
+
+    public class WeatherForecastController : TestODataController
+    {
+        private static readonly string[] Summaries = new[]
```

Comment:
>Summaries [](start = 41, length = 9)

Is this used anywhere?

Reply:
No, Can we leave it unchanged, later, if we add more test cases, we can re-use it?

---
In reply to: [485246056](https://github.com/OData/WebApi/pull/2267#discussion_r485246056) [](ancestors = 485246056)